docs/architecture/modern-web-apps-azure/develop-asp-net-core-mvc-apps.md

```diff
@@ -356,6 +435,72 @@ It's important that UseIdentity appear before UseMvc in the Configure method. Wh
 
 You can learn more about [configuring two-factor authentication](/aspnet/core/security/authentication/2fa) and [enabling external login providers](/aspnet/core/security/authentication/social/) from the official ASP.NET Core docs.
 
+### Authentication
+
+Authentication is the process of determining who is accessing the system. If you're using ASP.NET Core Identity and the configuration methods shown in the previous section, it will automatically configure some authentication defaults in the application. However, you can also configure these defaults manually, or override the ones set by AddIdentity. If you're using Identity, it configure cookie-based authentication as the default *scheme*.
+
+In web-based authentication, there are typically up to 5 actions that may be performed in the course of authenticating a client of a system. These are:
+
+- Authenticate. Use information provided by the client to create an identity for them to use within the application.
+- Challenge. This action is used to require the client to identify themselves.
+- Forbid. Inform the client they are forbidden from performing an action.
+- Sign-in. Persist the existing client in some way.
+- Sign-out. Remove the client from persistence.
+
+There are a number of common techniques for performing authentication in web applications. These are referred to as schemes. A given scheme will define actions for some or all of the above options. Some schemes only support a subset of actions, and may require a separate scheme to perform those it does not support. For example, the OpenId-Connect (OIDC) scheme doesn't support Sign-in or Sign-out, but is commonly configured to use Cookie authentication for this persistence.
+
+In your ASP.NET Core application, you can configure a `DefaultAuthenticateScheme` as well as optional specific schemes for each of the actions described above. For example, `DefaultChallengeScheme`, `DefaultForbidScheme`, etc. Calling [`AddIdentity<TUser,TRole>`](https://github.com/dotnet/aspnetcore/blob/release/3.1/src/Identity/Core/src/IdentityServiceCollectionExtensions.cs#L38-L102) configures a number of aspects of the application and adds many required services. It also includes this call to configure the authentication scheme:
+
+```csharp
+services.AddAuthentication(options =>
+{
+    options.DefaultAuthenticateScheme = IdentityConstants.ApplicationScheme;
+    options.DefaultChallengeScheme = IdentityConstants.ApplicationScheme;
+    options.DefaultSignInScheme = IdentityConstants.ExternalScheme;
+});
+
+These schemes use cookies for persistence and redirection to login pages for authentication by default. These schemes are appropriate for web applications that interact with users via web browsers, but not recommended for APIs. Instead, APIs will typically use another form of authentication, such as JWT bearer tokens.
```

Comment:
The statements are merged into code section. Looks like it should be a different para, @ardalis ?